server/src/test/java/org/elasticsearch/action/admin/cluster/allocation/ClusterAllocationExplainRequestTests.java

```diff
@@ -37,4 +37,56 @@ public void testSerialization() throws Exception {
         assertEquals(request.getCurrentNode(), actual.getCurrentNode());
     }
 
+    public void testToStringWithEmptyBody() {
+        ClusterAllocationExplainRequest clusterAllocationExplainRequest = new ClusterAllocationExplainRequest(randomTimeValue());
+        clusterAllocationExplainRequest.includeYesDecisions(true);
+        clusterAllocationExplainRequest.includeDiskInfo(false);
+
+        String expected = "ClusterAllocationExplainRequest[useAnyUnassignedShard=true,"
+            + "include_yes_decisions?=true,include_disk_info?=false";
+        String actual = clusterAllocationExplainRequest.toString();
+
+        assertTrue(actual.startsWith(expected));
```

Comment:
Should this be an assertEquals? It looks like you've spelled out the entire string, not just the beginning?